sys/kern/fdt.c

```diff
@@ -236,8 +240,10 @@ int FDT_get_reserved_mem(fdt_mem_reg_t *mrs, size_t *cntp) {
        child = FDT_peer(child)) {
     if (cnt == FDT_MAX_RSV_MEM_REGS)
       return ERANGE;
+#if 0
     if (FDT_hasprop(child, "no-map"))
       continue;
+#endif
```

Comment:
dead code?

Reply:
Possibly. I want @MichalBlk to comment on it.

Reply:
This code should be removed.

Reply:
Done!